packit/config.py

```diff
@@ -130,6 +131,7 @@ def get_from_dict(cls, raw_dict: dict, validate=True) -> "Config":
         config.github_app_id = raw_dict.get("github_app_id", "")
         config.github_app_cert_path = raw_dict.get("github_app_cert_path", "")
         config.webhook_secret = raw_dict.get("webhook_secret", "")
+        config.actions_environment = raw_dict.get("openshift", "cli")
```

Comment:
you renamed the variable, but in config it's still called openshift; as for the name, how about `actions_handler`?

Reply:
Well, I would rename it to "cli" to "local" as you propose bellow and "openshift" to "sandcastle" because of we are going to use it.

Reply:
I mean, you read key openshift from the config, that's what I'm taking about.

Reply:
openshift -> actions_handler done.